index.html

```diff
@@ -39,18 +39,36 @@ <h3 class="logo__title"><a href="#" class="link__logo">My Logo</a></h3>
         <i class="fas fa-bars fa-lg menu--humb"></i>
         <ul class="cont-menu-list">
           <li class="menu-list">
-            <a href="#" class="link-menu">Portfolio</a>
+            <a href="#portfolio" class="link-menu">Portfolio</a>
           </li>
           <li class="menu-list">
-            <a href="#" class="link-menu">About</a>
+            <a href="#about" class="link-menu">About</a>
           </li>
           <li class="menu-list">
-            <a href="#" class="link-menu">Contact</a>
+            <a href="#contact" class="link-menu">Contact</a>
           </li>
         </ul>
       </nav>
     </header>
 
+    <nav class="mobile-menu">
+      <button type="button" class="dis-mobile-menu">
+        <i class="fas fa-times fa-2x"></i>
+      </button>
+
+      <ul class="cont-mobile-menu-list">
+        <li class="menu-mobile-list">
+          <a href="#portfolio" class="link-mobile-menu">Portfolio</a>
+        </li>
+        <li class="menu-mobile-list">
+          <a href="#about" class="link-mobile-menu">About</a>
+        </li>
+        <li class="menu-mobile-list">
+          <a href="#contact" class="link-mobile-menu">Contact</a>
+        </li>
+      </ul>
+    </nav>
```

Comment:
Your _mobile menu_ does not match the provided [desgn template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). We actually want our design to as much as possible closely mimic the design of the provided template. Kindly ensure to improve these aspects:
- [ ] The background color of the overlay should match the Figma template.
- [ ] The background should be blurred so that the contents on the menu overlay become clearer to read. This can be achieved by using the _CSS rgba_ color property. This means, to achieve a blurred effect the alpha value should be high and vice versa.
- [ ] The hamburger and the close button should be at the same position, but only interchangeable

**Your mobile menu design:**
![image](https://user-images.githubusercontent.com/12745474/136086926-30b5addd-6951-45da-9b77-0715ad3ea39a.png)


**The original menu design:**
![image](https://user-images.githubusercontent.com/12745474/136086676-5cbf5d80-f223-4e3f-b37d-18215d493fd3.png)